packages/llm/src/utils/tools.ts

```diff
@@ -0,0 +1,59 @@
+import { Api, KnowChainId } from "@polkadot-agent-kit/common"
+import { convertAddress, toMultiAddress } from "@polkadot-agent-kit/core"
+import { MultiAddress } from "@polkadot-api/descriptors"
+import { ToolResponse, ChainNotAvailableError, InvalidAddressError } from "../types"
+
+// Utility function to generate tool_call_id
```

Comment:
```suggestion
/* Utility function to generate `tool_call_id` */
```
I don't see any reference to `tool_call_id` which makes the doc not valuable to the maintainers